app/views/layouts/_footer.html.erb

```diff
@@ -8,8 +8,15 @@
       <p>
         Having trouble? <a href="http://github.com/publiclab/mapknitter/issues">Submit bugs here</a>.
       </p>
-        <p>
-          MapKnitter uses <a href="http://gdal.org">GDAL</a>, <a href="http://leafletjs.com">Leaflet</a>, <a href="http://rubyonrails.org">Ruby on Rails</a>, <a href="http://imagemagick.org">ImageMagick</a>, and other open source software, and makes extensive use of the open source font <a href="http://www.theleagueofmoveabletype.com/junction">Junction</a>.</p>
+         <p>
++       MapKnitter uses <a href="http://gdal.org">GDAL</a>, <a href="http://leafletjs.com">Leaflet</a>, <a href="http://rubyonrails.org">Ruby on Rails</a>, <a href="http://imagemagick.org">ImageMagick</a>, and other open source software, and makes extensive use of the open source font <a href="http://www.theleagueofmoveabletype.com/junction">Junction</a>.
```

Comment:
I don't like the idea of having this in one huge line, but let's wait on someone elses review.

Reply:
Hi @alaxalves. What would be your preferred solution, because it does look good to me

Reply:
Just break the lines, that's all.